frontend/src/components/search/SearchFilterAccordion/SearchFilterToggleAll.tsx

```diff
@@ -11,15 +13,23 @@ const SearchFilterToggleAll: React.FC<SearchFilterToggleAllProps> = ({
     <div className="grid-col-fill">
       <button
         className="usa-button usa-button--unstyled font-sans-xs"
-        onClick={onSelectAll}
+        onClick={(event) => {
+          // form submission is done in useSearchFilter, so
+          // prevent the onClick from submitting here.
+          event.preventDefault();
+          onSelectAll?.();
+        }}
       >
         Select All
       </button>
     </div>
     <div className="grid-col-fill text-right">
       <button
         className="usa-button usa-button--unstyled font-sans-xs"
-        onClick={onClearAll}
+        onClick={(event) => {
+          event.preventDefault();
+          onClearAll?.();
+        }}
```

Comment:
`select all`/`clear all` options need to prevent form submit, that should be done in `useSearchFilter` at the same time as query param update